docker/dockerfiles/Dockerfile_commcarehq

```diff
@@ -9,16 +9,39 @@ RUN requirements/uninstall-requirements.sh
 COPY requirements/requirements.txt /vendor/requirements.txt
 COPY requirements/dev-requirements.txt /vendor/dev-requirements.txt
 COPY requirements/test-requirements.txt /vendor/test-requirements.txt
+COPY requirements/prod-requirements.txt /vendor/prod-requirements.txt
 
 RUN pip install \
     -r /vendor/requirements.txt \
     -r /vendor/dev-requirements.txt \
+    -r /vendor/prod-requirements.txt \
     --user --upgrade \
     && rm -rf /root/.cache/pip
 
+RUN npm install -g less@1.3.1
+
+WORKDIR /opt
+
+RUN git clone https://github.com/less/less.js.git lessc
+
+WORKDIR /opt/lessc
+
+RUN git reset --hard 546bedd3440ff7e626f629bef40c6cc54e658d7e
+
 WORKDIR /mnt
 
+RUN wget -O jython_installer-2.5.3.jar http://search.maven.org/remotecontent?filepath=org/python/jython-installer/2.5.3/jython-installer-2.5.3.jar 
+RUN ls
+RUN mkdir /usr/local/lib/jython
+RUN java -jar jython_installer-2.5.3.jar -s -d /usr/local/lib/jython
+RUN ln -s /usr/local/lib/jython/bin/jython /usr/local/bin/
+RUN wget http://peak.telecommunity.com/dist/ez_setup.py
+RUN jython ez_setup.py
```

Comment:
Similar to above, this can be turned into a single multline command.

It should also do some cleanup like removing the installers